server

```diff
@@ -195,7 +195,7 @@ setup_install() {
 get_full_version() {
   PATCH_VERSION=$(echo $SCYLLA_VERSION | awk -v FS='.' '{print $3}')
  if [ -n "$PATCH_VERSION" ] && [ -z "$DEFAULT_SCYLLA_VERSION" ] || [[ $SCYLLA_VERSION == *rc* ]]; then
-    FULL_SCYLLA_VERSION=$(apt-cache madison ${SCYLLA_PRODUCT} | grep -F $SCYLLA_VERSION | cut -d'|' -f 2 | sed 's/[[:space:]]//g')
+    FULL_SCYLLA_VERSION=$(apt-cache madison ${SCYLLA_PRODUCT} | grep -F $SCYLLA_VERSION | cut -d'|' -f 2 | sed 's/[[:space:]]//g' | head -n1)
```

Comment:
In this case, I think check of `PATCH_VERSION` is not required anymore.

Reply:
Actually it is needed, when trying to install `nightly-master`